test-service/src/ts_sdk_config_params.erl

```diff
@@ -234,3 +254,49 @@ add_tag_application_version(_, Options) -> Options.
 
 application_or_empty(#{application := Application} = _Options) -> Application;
 application_or_empty(_Options) -> #{}.
+
+basic_tls_options(Options) ->
```

Comment:
The erlang SDK does not have a basic TLS configuration depending on if the underlying OTP has a default configuration. So we set one here to ensure we are always using secure defaults for the contract tests.